scripts/code_style.py

```diff
@@ -102,7 +102,7 @@ def get_uncrustify_version() -> str:
     else:
         return str(result.stdout, "utf-8")
 
-def check_style_is_correct(src_file_list: List[str]) -> bool:
+def check_style_is_correct(src_file_list: List[str], use_git_diff: bool, use_pager: bool) -> bool:
```

Comment:
`check_python_files` is failing because this function is called with one argument in line 174. Please add extra arguments or (even better) set these to have default values:
```suggestion
def check_style_is_correct(src_file_list: List[str], use_git_diff: bool = False, use_pager: bool = False) -> bool:
```